cmd/analyze/main.go

```diff
@@ -176,50 +177,51 @@ func main() {
 	flag.Parse()
 
 	if err := featureflags.Update(*features); err != nil {
-		slog.Error("Failed to parse flags", "error", err)
-		return
+		return -1, fmt.Errorf("failed to parse flags: %w", err)
```

Comment:
It's unusual to see an exit code that is negative. The codes I'm familiar with are 0 for ok, 1 for failure, and 2 for bad usage. You can invent more, but I discourage it unless you have a really compelling reason.

Reply:
That's a good convention. I'm not quite sure where I got -1 (255) from. I will change it to 2.